components/new-homepage/TheFutureIsCollectiveSection.js

```diff
@@ -0,0 +1,141 @@
+
+  return (
+    <Fragment>
+      <Flex justifyContent="center" alignItems="center" px="16px" mt="40px">
+        <Flex flexDirection="column" alignItems="center">
+          <Box>
+            <H1
+              letterSpacing={['-0.04em', '0.012em']}
+              fontSize={['52px', '64px', '72px']}
+              lineHeight={['56px', '80px', '88px']}
+              textAlign="center"
+              color="primary.900"
+              fontWeight={['700', '900']}
+            >
+              <FormattedMessage id="home.futureIsCollective" defaultMessage={'The future is collective'} />
+            </H1>
+          </Box>
+          <Box my={[4, '40px']} maxWidth={['288px', '768px', '780px']}>
+            <P
+              fontSize={['16px', '20px', null, null, '18px']}
+              letterSpacing={[null, '-0.008em', null, null, '-0.2px']}
+              lineHeight={['24px', '28px', null, null, '27px']}
+              textAlign="center"
+              color="black.800"
+              fontWeight="500"
+            >
+              <FormattedMessage
+                id="home.futureIsCollective.description"
+                defaultMessage={
+                  'Open Collective is a <b>legal and financial</b> toolbox for grassroots groups. It’s a <b>fundraising + legal status + money management</b> platform for your community. What do you want to do?'
+                }
+                values={{
+                  b: I18nBold,
+                }}
+              />
+            </P>
+          </Box>
+          <Box mb="40px" display="flex" flexDirection={['column', 'row']} alignItems="center">
+            <Link href="/create">
+              <StyledButton
+                width={['288px', '125px']}
+                my={['12px', null, 0]}
+                mr={[0, '24px']}
+                buttonStyle="marketing"
+                whiteSpace="nowrap"
+                backgroundColor="primary.900"
+                fontSize="16px"
+                lineHeight="20px"
+              >
+                <FormattedMessage id="home.startNow" defaultMessage="Start Now" />
+              </StyledButton>
+            </Link>
+            <StyledButton
+              onClick={() => setShowModal(true)}
+              my={['12px', null, 0]}
+              width={['288px', '196px']}
+              fontSize="16px"
+              lineHeight="20px"
+            >
+              <Span mr={'14px'} textDecoration="underline">
+                <FormattedMessage id="home.makeCommunitySection.watchVideo" defaultMessage="Watch Video" />
+              </Span>
+              <Span className="arrowIcon">
+                <RightArrow size="15" />
+              </Span>
+            </StyledButton>
+          </Box>
+          <Box display={[null, 'none']}>
+            <NextIllustration
+              width={320}
+              height={589}
+              alt="The future is collective."
+              src="/static/images/new-home/budget-illustration-xs.png"
```

Comment:
@flickz What's the purpose of providing multiple versions for the same image? Isn't next image automatically generating different resolutions?